openmdao/recorders/web_recorder.py

```diff
@@ -456,3 +406,9 @@ def convert_to_list(self, obj):
             return []
         else:
             return obj
+
+    def close(self):
+        """
+        Close.
+        """
+        pass
```

Comment:
The base class version that was there before WAS vestigial... the sqlite_recorder version disconnects from the DB...so yes, I should have, instead of deleting the base version, make it throw a not implemented?